diff --git a/node_modules/postcss-scrollbar/dist/index.js b/node_modules/postcss-scrollbar/dist/index.js
index a569d58..68ae07f 100644
--- a/node_modules/postcss-scrollbar/dist/index.js
+++ b/node_modules/postcss-scrollbar/dist/index.js
@@ -84,18 +84,32 @@ function postcssScrollbar({
       });
     }
 
-    let values = nodes.filter(value => value.type === 'word').reduce((acc, curr, idx) => {
+    function getColorValue(node) {
+      if (colorMap[node.value]) {
+        return colorMap[node.value]
+      }
+
+      if (node.type === 'word') {
+        return node.value
+      }
+
+      if (node.type === 'function') {
+        return decl.value.slice(node.sourceIndex, node.sourceEndIndex)
+      }
+    }
+
+    let values = nodes.filter(value => value.type === 'word' || (value.type === 'function' && value.value === 'rgb')).reduce((acc, curr, idx) => {
       if (idx >= 1) {
         return { ...acc,
-          track: colorMap[curr.value] || curr.value,
-          corner: colorMap[curr.value] || curr.value
+          track: getColorValue(curr),
+          corner: getColorValue(curr)
         };
       }
 
       return {
-        thumb: colorMap[curr.value] || curr.value,
-        track: colorMap[curr.value] || curr.value,
-        corner: colorMap[curr.value] || curr.value
+        thumb: getColorValue(curr),
+        track: getColorValue(curr),
+        corner: getColorValue(curr)
       };
     }, {});
     let {
